src/gui.cpp

```diff
@@ -0,0 +1,36 @@
+#include "gui.h"
+
+void GUI::clear_screen() {
+  // Mac/Linux
+#ifndef DEBUG
+  system("clear");
+#endif
+}
+
+void GUI::print_border() {
+  char border[BOARD_WIDTH + 1] = "|";
+  for (int i = 1; i < BOARD_WIDTH - 1; ++i) {
```

Comment:
nit - keep tidy types
```suggestion
  for (size_t i = 1; i < BOARD_WIDTH - 1; ++i) {
```

Reply:
Suppose I could change all my `int` to `size_t`. Is this usually the case and not even unsigned int? Extra memory?

Reply:
I don't care so much about extra memory, just being intentional about types and what they represent. To me `size_t` is the type for `sizeof`, array length, and array index